src/main/java/com/greenfox/dramacsoport/petclinicbackend/controllers/appUser/auth/LoginController.java

```diff
@@ -30,12 +31,7 @@ public ResponseEntity<?> loginUser(@Valid @RequestBody LoginRequestDTO requestDT
             return new ResponseEntity<>(bindingResult.getAllErrors(), HttpStatus.BAD_REQUEST);
         }
 
-        try {
-            logger.info("Login request successful: {}", requestDTO);
-            return new ResponseEntity<>(authService.login(requestDTO), HttpStatus.OK);
-        } catch (UsernameNotFoundException e) {
-            logger.error("Invalid credentials: {}", e.getMessage());
-            return new ResponseEntity<>("Bad credentials!", HttpStatus.FORBIDDEN);
-        }
+        logger.info("Login request successful: {}", requestDTO);
```

Comment:
Well please keep in mind, that you are exposing the raw password here. Which is not the nicest thing. You probably should not log the users password in raw format.